eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -98,7 +97,9 @@ jobs:
 
       - pwsh: |
           Write-Host "${{parameters.DocArgs}}"
-          node $(Build.SourcesDirectory)/eng/tools/generate-doc/index.js --dgOp "dg" -i "inc" --inc "${{parameters.DocArgs}}"
+          $docDirectory = ${{parameters.ServiceDirectory}}
```

Comment:
there is a chance this line needs quotes.

Reply:
So if we are adding quotes to the next line then we shouldn't add them here right? Or we should add them here and not on the next line?